svgsynoptic2/taurussynopticwidget.py

```diff
@@ -202,6 +218,8 @@ def attribute_listener(self, model, evt_src, evt_type, evt_value):
                              (model, value))
 
         elif isinstance(value, (bool, np.bool_)):
+            #Change value 
+            #value = self.set_custom_value('/'.join(model.split('/')[:-1]), value)
```

Comment:
I'm not sure why this is commented, but I noticed a problem when I have too complicated functions in some parts like this. It is more possible to get error `terminate called after throwing an instance of 'Tango::DevFailed'` when this is uncommented. When I removed inquiry to database: ` PyTango.get_device_proxy(model).info().dev_class:` and just checked `model` name for some simple schemes in name it was less possible to get such error.